### ISSUE TITLE
Allow other audio codecs to be used other than mp3

### ISSUE THREAD

**Issue by ldatmpacc** [ManimCommunity/manim-voiceover#60]
## Description of proposed feature

Currently, rendering to .webm doesn't work with manim-voiceover, as all audio files use .mp3's, which the WebM format does not support(it only supports Vorbis or OPUS).

The MP3 codec is also pretty old and really lossy, especially when compared to more modern formats like AAC, Vorbis or OPUS. On that ground, there should be a way for audio to be encoded as another format(which could be done with FFmpeg on the source files).

## How can the new feature be used?

It would re-enable WebM output without having to spend extra time reencoding(and losing fidelity), which some people might prefer over having to deal with over the patented H.264/H.265 found in .mp4 files.

## Additional comments

As I said before, it could easily be done using FFmpeg.

**Comment by osolmaz** [comment]
I will try and see if there is an easy way to do this in the next version.
